content/src/logic/deployments.ts

```diff
@@ -52,10 +53,14 @@ export async function retryFailedDeploymentExecution(
         )
       } catch (error) {
         // it failed again, override failed deployment error description
-        const errorDescription = error.message
-        components.failedDeploymentsCache.reportFailure({ ...failedDeployment, errorDescription })
+        const errorDescription = error.message + ''
 
-        logger.info(`Failed to fix deployment of entity`, { entityId, entityType })
+        // TODO [mendez] this condition has no test coverage
```

Comment:
I don't think this changes are necessary, read comments below

Reply:
I added it because I've encountered an infinite loop of retries while debugging.

Reply:
Aaah, maybe that's you are learning the new deployment from every other catalyst?